src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,8 +1,14 @@
 package mate.academy.spring.dto.request;
 
+import mate.academy.spring.validation.Email;
+import mate.academy.spring.validation.ValidPassword;
+
+@ValidPassword
 public class UserRequestDto {
+    @Email
     private String email;
     private String password;
+    private String repeatPassword;
```

Comment:
As far as I see you don't have any check for password except null and equals with repeatPassword.
Please enhance your PasswordValidator or add annotations to have reasonable checks for password length

Reply:
Done